### PR TITLE
[ONNX] Add com.microsoft.gelu operator support into frontend/onnx

### PR DESCRIPTION
**Pull request by MyGInt**
### Details:

- Add onnx op com.microsoft.gelu support in frontend/onnx
- Add gelu test in layer_tests

### Tickets:
 #11881

### CODE REVIEW

File: src/core/tests/models/onnx/com.microsoft/gelu.prototxt [hunk 1]
```diff
@@ -0,0 +1,46 @@
+ir_version: 3
```
Comment (t-jankowski) [inline src/core/tests/models/onnx/com.microsoft/gelu.prototxt:1]: Why the oldest `ir_version`? Let's use the newest supported by OV. Same about `opset_import.version`.

**Review by t-jankowski — CHANGES_REQUESTED** [review]

File: src/frontends/onnx/frontend/src/op/com.microsoft/gelu.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "op/com.microsoft/bias_gelu.hpp"
```
Comment (tomdol) [inline src/frontends/onnx/frontend/src/op/com.microsoft/gelu.cpp:6]:
```suggestion
```

**Review by tomdol — CHANGES_REQUESTED** [review]

**Review by t-jankowski — APPROVED** [review]

**Comment by t-jankowski** [comment]
@MyGInt Code Style check fails - `make clang_format_fix_all` should help.

**Review by tomdol — APPROVED** [review]

**Review by artkuli — APPROVED** [review]

**Comment by MyGInt** [comment]
> The code in `ops_bridge.cpp` is not correctly formatted. Please run `make clang_format_fix_all` locally in your build directory and commit the resulting changes.

 It works,  thanks a lot!

**Comment by tomdol** [comment]
There's a problem with one of the commits, GH is not going to accept it. Could you perhaps squash all of the commits in this PR under your username and force-push? This should resolve the issue.

**Pull request closed without merging**